### PR TITLE
Feature/boas 681 unpublish case

### PR DESCRIPTION
**Pull request by HadrianHughesKC**
## Describe your changes

* feat(api/applications): add route and controller for unpublishing case
* feat(api/applications): call unpublish endpoint from the form
* feat(api/applications): create a table to store case published states
* feat(api/applications): perform db mutation to unpublish case
* refactor(api/applications): change publishCase to use new table
* refactor(api/applications): update existing code to expect new table
* chore(api/applications): remove test for web unpublish page navigation
* chore(api/applications): regenerate Swagger docs

## Issue ticket number and link

[BOAS-681](https://pins-ds.atlassian.net/browse/BOAS-681)

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

[BOAS-681]: https://pins-ds.atlassian.net/browse/BOAS-681?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/web/src/server/applications/case/__tests__/applications-case.test.js [hunk 1]
```diff
@@ -222,13 +222,5 @@ describe('Applications case pages', () => {
 				expect(element.innerHTML).toContain('Unpublish the project');
 			});
 		});
-
-		describe('POST /case/123/unpublish', () => {
```
Comment (lutamKC) [inline apps/web/src/server/applications/case/__tests__/applications-case.test.js:226]: why was this removed?

Comment (HadrianHughesKC) [reply apps/web/src/server/applications/case/__tests__/applications-case.test.js]: It was failing for a weird reason. I've added it back in and fixed the issue now.

**Review by lutamKC — APPROVED** [review]
